rasa/core/lock_store.py

```diff
@@ -190,11 +198,17 @@ def __init__(
         db: int = 1,
         password: Optional[Text] = None,
         use_ssl: bool = False,
+        socket_timeout: float = DEFAULT_SOCKET_TIMEOUT_IN_SECONDS,
     ):
         import redis
 
         self.red = redis.StrictRedis(
-            host=host, port=int(port), db=int(db), password=password, ssl=use_ssl
+            host=host,
+            port=int(port),
+            db=int(db),
+            password=password,
+            ssl=use_ssl,
+            socket_timeout=socket_timeout,
```

Comment:
optional: outside of the scope of this PR, but it would be great if we could add a class description with all attributes or a docstring for the `__init__` method